requirements.prod.in

```diff
@@ -4,6 +4,7 @@
 # pip-compile --generate-hashes --output-file=requirements.prod.txt requirements.prod.in
 
 mkdocs
-mkdocstrings
 mkdocs-material
+https://github.com/opensafely-core/mkdocs-opensafely-backend-contracts/archive/bf1a19894a9d37c6673f02c7cdfd89e6a72e3101.zip#mkdocs-opensafely-backend-contracts
+mkdocstrings
```

Comment:
Will we have to manually update this?

Reply:
Yep, we haven't found a nicer way to do this without releasing to PyPI yet.  Since this is an internal plugin the theory is this won't be an issue 🤞

Reply:
A helper like the one that updates the `databuilder` package might be nice, but not really needed now.